Install instruction breaks because of libfreenect2's changes

About: https://github.com/code-iai/iai_kinect2#install

no `examples/protonect` in libfreenect2.
https://github.com/OpenKinect/libfreenect2/tree/master/examples

And header files are not found,
```
% mkdir build
% cd build
% cmake ../examples
-- The C compiler identification is GNU 4.8.4
-- The CXX compiler identification is GNU 4.8.4
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.26")
-- checking for module 'glfw3'
--   found glfw3, version 3.0.4
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found X11: /usr/lib/x86_64-linux-gnu/libX11.so
-- Found OpenGL: /usr/lib/x86_64-linux-gnu/libGL.so
-- Configuring done
-- Generating done
-- Build files have been written to: /home/wkentaro/tech/libfreenect2/build
% make
Scanning dependencies of target Protonect
[ 33%] Building CXX object CMakeFiles/Protonect.dir/Protonect.cpp.o
/home/wkentaro/tech/libfreenect2/examples/Protonect.cpp:36:33: fatal error: libfreenect2/logger.h: No such file or directory
 #include <libfreenect2/logger.h>
                                 ^
compilation terminated.
make[2]: *** [CMakeFiles/Protonect.dir/Protonect.cpp.o] Error 1
make[1]: *** [CMakeFiles/Protonect.dir/all] Error 2
make: *** [all] Error 2
```

--- a/README.md
+++ b/README.md
@@ -113,7 +113,7 @@ sudo dpkg -i libglfw3*_3.0.4-1_*.deb
 cd ..
 mkdir build
 cd build
-cmake ../examples/protonect/ -DENABLE_CXX11=ON
+cmake .. -DENABLE_CXX11=ON
 make && sudo make install
 ```
 

